run-your-c++-code-on-the-web/README.md

```diff
@@ -19,7 +19,7 @@ how to [deal with long running tasks](yetanotherblog).
 So today's aim is to have a simple web app that determines the root of a mathematical function _2x^3 - 4x^2 + 6_, i.e. the
 value of _x_ where _y = 0_.
 
-![equation.svg.png](equation.svg.png)
+![equation.svg](equation.svg)
```

Comment:
does medium support svg?

Reply:
doh, no it does not.

Reply:
See https://github.com/NLESC-JCER/run-cpp-on-web/issues/44